docs/content/integrations/looker.mdx

```diff
@@ -0,0 +1,109 @@
+---
+title: "Using Dagster with the Looker API"
+description: Represent your Looker assets in Dagster
+---
+
+# Using Dagster with Looker
+
+<ExperimentalCallout />
+
+This guide provides instructions for using Dagster with Looker. Your Looker assets, such as views, explores, and dashboards, can be represented in the Dagster asset graph, allowing you to track lineage and dependencies between Looker assets and upstream data assets you are already modeling in Dagster. You can also use Dagster to orchestrate Looker PDTs, allowing you to trigger refreshes of these materialized tables on a cadence or based on upstream data changes.
+
+## What you'll learn
+
+- How to represent Looker assets in the Dagster asset graph, including lineage to other Dagster assets.
```

Comment:
Are we covering lineage to other Dagster assets in this guide? It looks like that section might be missing.

Reply:
That can be done in a separate PR

Reply:
That makes sense. Let's just remove "including lineage to other Dagster assets" for now.